src/PiGui.cpp

```diff
@@ -336,6 +336,13 @@ void PiGui::EndFrame() {
 }
 
 void PiGui::NewFrame(SDL_Window *window) {
+	// Ask ImGui to hide OS cursor if GUI is not being drawn:
+	// it will do this if MouseDrawCursor is true. After the frame
+	// is created, we set the actual cursor draw state.
+	if (!Pi::DrawGUI)
+	{
+		ImGui::GetIO().MouseDrawCursor = true;
```

Comment:
No. It's reversed here, it means that ImGui is responsible for the pointer and not the OS. See `ImGui_*_NewFrame` implementations.